### PR TITLE
Add postgres service for test flow

### PR DESCRIPTION
**Pull request by KaylaBrady**
Ticket: https://app.asana.com/0/1199987702860136/1200240419411845
This PR adds a postgres service container to the test workflow. Without a postgres db, the tests cannot be run. 

I tested this change locally using https://github.com/nektos/act. However, I found there was an [issue](https://github.com/nektos/act/issues/173) where services did not work when testing locally . Followed [this](https://mauricius.dev/run-and-debug-github-actions-locally/) tutorial to start a postgres container locally for testing and confirmed I was able to get the tests to run from this workflow. 
image

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -25,7 +25,7 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
-  config :ueberauth, Ueberauth,
+config :ueberauth, Ueberauth,
```
Comment (KaylaBrady) [inline config/config.exs:28]: cleanup caught in `mix format`

**Comment by paulswartz** [comment]
This looks good: can you re-enable the action so we can see it running in GA?

**Comment by KaylaBrady** [comment]
@paulswartz actions ran successfully!

**Review by paulswartz — APPROVED** [review]
🍰

**Pull request merged**